source/indexes.txt

```diff
@@ -164,8 +162,8 @@ Atlas Search Index Management
 The following sections contain code examples that describe how to manage Atlas Search
 indexes.
 
-.. TODO: To learn more about Atlas search indexes, see the :ref:`kotlin-sync-atlas-search-index`
-.. guide.
+To learn more about Atlas search indexes, see the :ref:`kotlin-sync-atlas-search-index`
```

Comment:
S: I think "search" should be capitalized here
```suggestion
To learn more about Atlas Search indexes, see the :ref:`kotlin-sync-atlas-search-index`
```

Reply:
Applies to all mentions of "Atlas search index"

Reply:
Good catch!